app/presenters/migration/index.rb

```diff
@@ -28,7 +67,7 @@ def page_content_heading
     protected
 
     def t(*args, **options)
-      I18n.t(*args, options.reverse_merge(scope: 'site.campaigns.migration.pages.index'))
+      I18n.t(*args, options.reverse_merge(scope: 'contribute.campaigns.migration'))
```

Comment:
The translations for this page should be moved under `contribute.campaigns.migration.pages.index`

Reply:
I've moved the keys under that namespace